clowdapp.yaml

```diff
@@ -11,6 +11,7 @@ objects:
   spec:
     envName: ${ENV_NAME}
     dependencies:
+    - host-inventory
```

Comment:
Could you add `ingress` as well

Reply:
I think host-inventory has ingress as a dependency... In the log, I can see ingress is deployed.